test/JDBC/expected/BABEL-CROSS-DB-vu-prepare.out

```diff
@@ -263,3 +263,19 @@ GO
 
 GRANT EXECUTE ON FUNCTION master_dbo.babel_cross_db_vu_prepare_pg_func to master_babel_cross_db_vu_prepare_u1;
 GO
+
```

Comment:
can we have a test case where we revoke connect from user and then login tries to connect to a database which has guest role enabled.

Reply:
We already have such test cases where these is guest enabled on the database. If we revoke connect from the user,  login will not be able to establish connection to the database.

Reply:
But login should still be able connect if guest user is enabled. I was searching for that test, couldn't find one.